MediaBrowser.Controller/Plugins/IPluginConfigurationPage.cs

```diff
@@ -3,36 +3,6 @@
 
 namespace MediaBrowser.Controller.Plugins
 {
-    /// <summary>
-    /// Interface IConfigurationPage.
-    /// </summary>
-    public interface IPluginConfigurationPage
-    {
-        /// <summary>
-        /// Gets the name.
-        /// </summary>
-        /// <value>The name.</value>
-        string Name { get; }
-
-        /// <summary>
-        /// Gets the type of the configuration page.
-        /// </summary>
-        /// <value>The type of the configuration page.</value>
-        ConfigurationPageType ConfigurationPageType { get; }
-
-        /// <summary>
-        /// Gets the plugin.
-        /// </summary>
-        /// <value>The plugin.</value>
-        IPlugin Plugin { get; }
-
-        /// <summary>
-        /// Gets the HTML stream.
-        /// </summary>
-        /// <returns>Stream.</returns>
-        Stream GetHtmlStream();
-    }
-
     /// <summary>
     /// Enum ConfigurationPageType.
     /// </summary>
```

Comment:
File should be renamed to reflect the type in it.

Reply:
Removed the file. Did a bit more checking. 

ConfigurationPageType was only ever checked in the server API controller. Not referred to in any of the plugins on jellyfin.

Web side it seemed to be a hardcoded value, which is the default, and was optional.